### ISSUE TITLE
[A11Y] Missing text alternative

### ISSUE THREAD

**Issue by erikkroes** [Tiqr/eduid-app-ios#256]
Like https://github.com/OpenConext/OpenConext-myconext/issues/1062 and https://github.com/Tiqr/eduid-app-android/issues/263

## Finding
This image is a visual indicator and has no text alternative

<img width="99" height="115" alt="" src="https://github.com/user-attachments/assets/5f742319-0c58-46a6-abd3-a14dfb09b4fb" />

## Location

This icon is used on the activity-page.

<img width="831" height="649" alt="" src="https://github.com/user-attachments/assets/7ca95f04-7ba1-4580-81ed-3dd0023dff40" />

## Problem and solution

Not everybody can see images, some people depend on alternatives. These alternatives should be sufficiently valuable that they could replace the image.

When an image communicates something (for example, when it has text), make sure there is a text alternative. Any text in the image should be part of the alternative.
In this case the alternative could be something like "has access to your information"

## Resources
- https://www.erikkroes.nl/glossary/#accessible-name
- https://www.erikkroes.nl/glossary/#visually-hidden
- https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html
- https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
- https://www.a11yproject.com/posts/how-to-hide-content/
